### PR TITLE
BaSP-MR-235: add select input in forms Member

### PR DESCRIPTION
**Pull request by Emanuel-Lamberti**
The normal input was replaced with a select input with options in the forms to create Members.

### CODE REVIEW

**Review by FrancoLelli — APPROVED** [review]

File: src/Components/Auth/SignUp/index.jsx [hunk 1]
```diff
@@ -134,6 +136,8 @@ const SignForm = () => {
     }
   };
 
+  const memberships = ['Classic', 'Black', 'Only_classes'];
+
```
Comment (YhanahiRosenthal) [inline src/Components/Auth/SignUp/index.jsx:140]: you should just put 'Only classes'

Comment (Emanuel-Lamberti) [reply src/Components/Auth/SignUp/index.jsx:140]: all right

File: src/Components/Members/Form/MembersForm.js [hunk 2]
```diff
@@ -166,6 +173,8 @@ const MembersForm = () => {
     setModalAddConfirmOpen(true);
   };
 
+  const memberships = ['Classic', 'Black', 'Only_classes'];
+
```
Comment (YhanahiRosenthal) [inline src/Components/Members/Form/MembersForm.js:177]: here too

**Review by YhanahiRosenthal — APPROVED** [review]
Good job!

**Pull request merged**